graylog2-server/src/main/java/org/graylog2/cluster/certificates/CertificatesService.java

```diff
@@ -127,4 +127,14 @@ public Optional<String> readCert(final KeystoreMongoLocation keystoreMongoLocati
         return Optional.empty();
     }
 
+    public void removeCert(final KeystoreMongoLocation keystoreMongoLocation) {
```

Comment:
This method could potentially return `boolean` to indicate if smth was indeed removed (or `int` with a number of removed documents, as we tend to do in other services).